src/main/java/org/jabref/gui/openoffice/StyleSelectDialogViewModel.java

```diff
@@ -71,7 +71,7 @@ public void addStyleFile() {
                 styles.setAll(loadStyles());
                 selectedItem.setValue(getStyleOrDefault(stylePath));
             } else {
-                dialogService.showErrorDialogAndWait(Localization.lang("Invalid style selected"), Localization.lang("You must select a valid style file."));
+                dialogService.showErrorDialogAndWait(Localization.lang("Invalid style selected"), Localization.lang("You must select a valid style file. Your style is probably missing a line for the type \"default\"."));
```

Comment:
Is it possible to get more information (than a boolean) about why a style file is not valid? I guess the missing `default` may only be one of many reasons and so far the error message is not really helpful in diagnosing the problem.

Reply:
This is more or less the only reason. The other is that if you have no sections defined.
Other things (Layout parsing etc) are already handled with exceptions 
https://github.com/JabRef/jabref/blob/940ef9d3be3f3877035fbcf3da6b637b53944c40/src/main/java/org/jabref/logic/openoffice/OOBibStyle.java#L327-L331